### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA and Jaws are  not announcing the content present in "Settings" dialog sequentially

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1521]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins, and then Select Browse repositories  button.
4. In Settings window , Press "Insert+B" to read  the entire dialog then observe the NVDA and JAWS  Behavior

**Expected result:**
NVDA and Jaws should announce all the elements in sequential manner

- Settings  dialog 
- Search
- tree view
level 1  Appearance & Behavior  collapsed  1 of 9
Keymap  2 of 9  level 1
Editor  collapsed  3 of 9  level 1
 Plugins  selected  4 of 9  level 1
Version Control  collapsed  5 of 9  level 1
Build, Execution, Deployment  collapsed  6 of 9  level 1
Languages & Frameworks  collapsed  7 of 9  level 1
Tools  collapsed  8 of 9  level 1
Microsoft Tools  collapsed  9 of 9  level 1
- Search Plugins
-  Show:All plugins  button
- table
Sort by: name
All available plugins in the table
coverage
- Install JetBrains plugin...  button  alt+J
- Browse repositories...  button  alt+B
- Install plugin from disk...  button  alt+D
- Ok button
- Cancel  button
- Apply  button  alt+A

**Actual Result:**
**NVDA Behavior**: NVDA is announcing as
Settings  dialog  Show:
Sort by: name
Azure Toolkit for IntelliJ
Version: 3.3.0-2017.3
Check or uncheck a plugin to enable or disable it.
edit  Ctrl+F
Alt+F3
blank
tree view
level 1  Appearance & Behavior  collapsed  1 of 9
Keymap  2 of 9  level 1
Editor  collapsed  3 of 9  level 1
Plugins  selected  4 of 9  level 1
Version Control  collapsed  5 of 9  level 1
Build, Execution, Deployment  collapsed  6 of 9  level 1
Languages & Frameworks  collapsed  7 of 9  level 1
Tools  collapsed  8 of 9  level 1
Microsoft Tools  collapsed  9 of 9  level 1
Install JetBrains plugin...  button  alt+J
Browse repositories...  button  alt+B
Install plugin from disk...  button  alt+D
All plugins  button
Show:
edit
blank
table
**Jaws Behavior:**
Settings dialog
Show: Sort by: name Azure Toolkit for IntelliJVersion: 3.3.0-2017.3 Check or uncheck a plugin to enable or disable it. 
Show help contents Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![settings-insert-b](https://user-images.githubusercontent.com/22956130/36188909-56f6e926-1175-11e8-9ee9-3bcd8221a8cd.jpg)
![nvda is not announing the content sequentially in settings window](https://user-images.githubusercontent.com/22956130/36188916-62c997b2-1175-11e8-8e48-8942162a961b.jpg)

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749974
